tf_agents/environments/gym_wrapper.py

```diff
@@ -165,6 +165,8 @@ def __init__(self,
     self._action_spec = spec_from_gym_space(self._gym_env.action_space,
                                             spec_dtype_map, simplify_box_bounds,
                                             'action')
+    self._reward_spec = specs.BoundedArraySpec(
+        shape=(), dtype=np.float64, minimum=gym_env.reward_range[0], maximum=gym_env.reward_range[1], name='reward')
```

Comment:
do all gym_env provide a reward_range?
Fix code indentation issues and long lines..

Reply:
Yes, this is part of the API since may 2016 https://github.com/openai/gym/commit/99f40a2fc12ddbc8981c1dd60cf36552e058904e

I've fixed the long lines, but I'm not sure about indentation issue